packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -209,7 +214,7 @@ export class DatabricksDriver extends JDBCDriver {
       // common export bucket config
       bucketType:
         conf?.bucketType ||
-        getEnv('dbExportBucketType', { supported: ['s3', 'azure'], dataSource }),
+        getEnv('dbExportBucketType', { supported: ['s3', 'azure', 'gcs'], dataSource }),
```

Comment:
It's better to define a const for this:
```js
const SUPPORTED_BUCKET_TYPES = ['s3', 'gcs', 'azure'];
```
And use it instead of copy-pasting it every time.